ProcessesApi.Tests/V1/E2ETests/Steps/UpdateSoleToJointProcessSteps.cs

```diff
@@ -163,6 +163,20 @@ public async Task ThenTheProcessStateIsUpdatedToApplicationSubmitted(UpdateProce
             await CheckProcessState(request.Id, SoleToJointStates.ApplicationSubmitted, SoleToJointStates.DocumentChecksPassed).ConfigureAwait(false);
         }
 
+        public async Task ThenTheProcessStateIsUpdatedToTenureInvestigationFailed(UpdateProcessQuery request)
```

Comment:
Can we not refactor this so that is it one method rather than 3 different method?
Maybe something like:
` public async Task ThenTheProcessStateIsUpdatedToTenureInvestigation(UpdateProcessQuery request, string newState)
        {
            await CheckProcessState(request.Id, newState, SoleToJointStates.ApplicationSubmitted).ConfigureAwait(false);
        }`